apphub/instance_segmentation/solov2/solov2_torch.py

```diff
@@ -585,8 +585,8 @@ def get_estimator(data_dir=None,
     init_lr = 1e-2 / 16 * batch_size
     model = fe.build(model_fn=SoloV2, optimizer_fn=lambda x: torch.optim.SGD(x, lr=init_lr, momentum=0.9))
     network = fe.Network(ops=[
-        Normalize(inputs="image", outputs="image", mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
         Permute(inputs="image", outputs='image'),
```

Comment:
Now that permute is no longer necessary, we should remove the permute and use ChannelTranspose in Pipeline.